`make` fails if PATH includes whitespace

`make` fails if PATH includes whitespace as below.

```
$ echo $PATH
/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin:/Users/naoki/.krew/bin:/Users/naoki/go/bin:/usr/local/opt/gnu-tar/libexec/gnubin:/usr/local/opt/gnu-sed/libexec/gnubin:/Volumes/TOSHIBA EXT/data/Vagrant/.vagrant.d:/usr/local/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/Wireshark.app/Contents/MacOS:/usr/local/kubebuilder/bin

$ make
./travis/pre-commit.sh
env: EXT/data/Vagrant/.vagrant.d:/usr/local/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/Wireshark.app/Contents/MacOS:/usr/local/kubebuilder/bin:/Users/naoki/go/bin: No such file or directory
make: *** [pre-commit] Error 127
```

/assign

--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@
 
 MYGOBIN := $(shell go env GOPATH)/bin
 PATH := $(PATH):$(MYGOBIN)
-SHELL := env PATH=$(PATH) /bin/bash
+SHELL := env PATH="$(PATH)" /bin/bash
 
 .PHONY: all
 all: pre-commit

